src/main/java/io/github/nucleuspowered/nucleus/modules/scheduled/tasks/BroadcastTask.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.modules.scheduled.tasks;
+
+import com.google.common.collect.Lists;
+import com.google.inject.Inject;
+import io.github.nucleuspowered.nucleus.ChatUtil;
+import io.github.nucleuspowered.nucleus.internal.scheduled.ScheduledTask;
+import io.github.nucleuspowered.nucleus.modules.admin.config.AdminConfigAdapter;
+import io.github.nucleuspowered.nucleus.modules.admin.config.BroadcastConfig;
+import io.github.nucleuspowered.nucleus.util.DelayedTimeValue;
+import org.spongepowered.api.Game;
+import org.spongepowered.api.command.source.ConsoleSource;
+import org.spongepowered.api.scheduler.Task;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.channel.MessageChannel;
+
+import java.util.List;
+
+public class BroadcastTask extends ScheduledTask {
+
+    private String messageToBroadcast;
+    @Inject
+    private Game game;
+    @Inject
+    private AdminConfigAdapter adminConfigAdapter;
+    @Inject
+    private ChatUtil chatUtil;
+
+    public BroadcastTask(String name, String messageToBroadcast, DelayedTimeValue delayedTimeValue) {
+        super(name, delayedTimeValue);
+        this.messageToBroadcast = messageToBroadcast;
+    }
+
+    @Override
+    protected void runScheduledTask(Task task) {
+        BroadcastConfig bc = adminConfigAdapter.getNodeOrDefault().getBroadcastMessage();
```

Comment:
I haven't heard much back from the forums, but now thinking about it we should be able to do plaintext even with templates (although you get a slightly more configuration which I think is acceptable), so I'll go ahead and start down that path.